js/init.js

```diff
@@ -0,0 +1,7 @@
+(function($){
+  $(function(){
+
+    $('.sidenav').sidenav();
+
+  }); // end of document ready
+})(jQuery); // end of jQuery name space
```

Comment:
1. We do not have .sidenav class in index.html
2. Do we know the exact functionality of (function($){...})(jQuery) here?
I normally build from what I know, trimming down copied code to bare minimal. But if there's a specific reason why it's here, then please ignore my comment.